source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1571,15 +1571,24 @@ message TransformationLoad {
 
   // Transformation that adds an OpLoad instruction from a pointer into an id.
 
-  // The result of the load instruction
+  // The result of the load instruction.
   uint32 fresh_id = 1;
 
-  // The pointer to be loaded from
+  // The pointer to be loaded from.
   uint32 pointer_id = 2;
 
+  // True if and only if the load should be atomic.
+  bool is_atomic = 3;
+
+  // Memory scope for the load. Ignored unless the load is atomic.
+  uint32 memory_scope = 4;
```

Comment:
Remove this. If `is_atomic` is true then we want the `TransformationLoad` class to always add an `OpAtomicLoad` instruction with scope `Invocation`, so there is no need to store anything about the scope in the protobuf.

Reply:
Done.

Reply:
Sorry, please ignore this comment.